pan/types.pan

```diff
@@ -990,3 +990,8 @@ type valid_interface = string with {
     };
     false;
 };
+
+@documentation{
+    desc = CPU architectures understood by Quattor
+}
+type cpu_architecture = string with match (SELF,'i386|ia64|x86_64|sparc');
```

Comment:
should always be `^(...)$`; or release panc 10.4
also, rhel now supports a lot more arch, incl power and arm